cmd/neo4j-mcp/main.go

```diff
@@ -49,9 +54,20 @@ func main() {
 		log.Printf("Failed to create database service: %v", err)
 		return
 	}
+	isAura := strings.Contains(cfg.URI, "database.neo4j.io")
+	anService := analytics.NewAnalytics(MixPanelToken, MixPanelEndpoint, isAura)
+
+	if cfg.Telemetry == "false" || MixPanelEndpoint == "" || MixPanelToken == "" {
+		log.Println("Telemetry disabled.")
+		anService.Disable()
+	} else if cfg.Telemetry == "true" {
+		anService.Enable()
+		log.Println("Telemetry is enabled to help us improve the product by collecting anonymous usage data such as: tools in use, OS and CPU architecture.")
```

Comment:
```suggestion
		log.Println("Telemetry is enabled to help us improve the product by collecting anonymous usage data such as: tools being used, the operating system, and CPU architecture.")
```

Copying the wording from the README so it's consistent :)